discovery-infra/update_bm_inventory_cm.py

```diff
@@ -32,6 +32,10 @@
     ("AGENT_DOCKER_IMAGE", ""),
     ("KUBECONFIG_GENERATE_IMAGE", ""),
     ("BASE_DNS_DOMAINS", ""),
+    ("IMAGE_BUILDER", ""),
+    ("CONNECTIVITY_CHECK_IMAGE", ""),
+    ("HARDWARE_INFO_IMAGE", ""),
+    ("INVENTORY_IMAGE", ""),
```

Comment:
@eranco74 could you describe what is the role of INVENTORY_IMAGE ?

Reply:
It's the bm-inventory image.

Reply:
I think it's used for the versions API